### PR TITLE
Added Cucumber reports and Fetaure numbering parameter to the framework

### PR DESCRIPTION
**Pull request by amrutaben**
Please review the changes

### CODE REVIEW

**Comment by amrutaben** [comment]
Please review

File: features/continuous_integration.feature [hunk 1]
```diff
@@ -1,10 +1,10 @@
 @ci @readonly
-Feature: Continuous integration check of the project
+Feature: [WATER - <JIRA no.>] Continuous integration check of the project
```
Comment (timstone) [inline features/continuous_integration.feature]: These are to check that cucumber is working ok for CI so doesn't need a Jira reference

File: features/continuous_integration.feature [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
   As a contributor to this project
   I want to know a change I made hasn't broken it (though it may have failing tests)
   So that we have a reliable base on which to build our acceptance tests
 
-  Scenario: Check Quke is working
+  Scenario: [WATER - <JIRA no.> - AC1] Check Quke is working
```
Comment (timstone) [inline features/continuous_integration.feature]: These are to check that cucumber is working ok for CI so doesn't need a Jira reference

**Comment by amrutaben** [comment]
Please review

**Pull request merged**